components/brave_new_tab_ui/components/default/settings/index.ts

```diff
@@ -361,6 +362,7 @@ export const StyledWidgetToggle = styled<WidgetToggleProps, 'button'>('button')`
   border: none;
   margin-top: 8px;
   cursor: pointer;
+  height: 41px;
```

Comment:
Natural height when in the "Hide" state was already 41px, but when toggling to "Add", was bumped down to 40.6px, better to just keep this constant.